Add post hackathon 2021 winners (#899)
oktadeveloper/okta-blog: _source/_posts/2021-09-29-okta-hackathon-2021-winners.adoc

---
layout: blog_post
title: "Announcing the Okta Identity Early Access Hackathon Winners"

> author: brian-demers

Reviewer: ```suggestion
author: joyce-morrell
```
changed

> by: advocate

Author: ```suggestion
by: internal-contributor
```

communities: [security]
description: "The winners of the Okta Developer Virtual Hackathon for 2021 are in!"
tags: [hackathon, security, oie]
tweets:
- "Announcing the Okta Identity Hackathon Winners:
🏆 1st: GooseClip - peer to peer as a service
🏆 2nd: Bill Panda - keep track of recurring bills
🏆 3rd: App_builder_ - generate Okta starter templates in a few simple steps
And more..."
- "What do Bill Panda, App_builder_, Bubble Connect, and @GooseClip have in common? They are all winning projects in the Okta Identity Early Access Hackathon hosted by @devpost! Read all about it #oie"
image: blog/okta-hackathon-2021-winners/hackathon-social.jpg
type: awareness
---
:page-liquid:
:toc: macro

> Back in early August, in partnership with DevPost, we announced the "Okta Identity Early Access Hackathon" and invited developers everywhere to "build something awesome with the Okta Identity Engine (OIE) Limited GA release."

Reviewer: ```suggestion
Back in early August, in partnership with DevPost, we announced the *Okta Identity Early Access Hackathon*, and invited developers everywhere to "build something awesome with the Okta Identity Engine (OIE) Limited GA release."
```
changed challenge name to italics, and added a comma

**Description:** Application to bootstrap and configure Okta sample applications.

Folks from Okta's Developer Relations team love to build with and create tools that get working applications up and running quickly, like https://start.spring.io[Spring] / https://start.steeltoe.io/[Steeltoe] Initializers, https://code.quarkus.io/[`code.quarkus.io`], and the https://cli.okta.com[Okta CLI], so it wasn't a big leap for us to see potential in *App_ builder_*.

Sameera Perera, who developed *App_builder_*, was motivated by curiosity, and by the desire to share what he'd learned. With his collection of starter templates, Sameera succeeds magnificently in his goal of "facilitating an easiest path for new users who need a lot of references to build up their first project." We can't wait to see what new templates are next.

++++
<div style="text-align: center; margin-bottom: 1.25rem">
<iframe width="700" height="394" style="max-width: 100%" src="https://www.youtube.com/embed/a9gfivJ8wHU" title="Template Application Builder" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
</div>
++++

== Second Place

> **Name:** https://devpost.com/software/family-newsletter[BillPanda] +

Reviewer: ```suggestion
**Name:** https://devpost.com/software/family-newsletter[Bill Panda] +
```
two words, not camel case.

**Prize:** $5,000 USD +
**Description:** Personal bill payment reminder system.

> BillPanda is one of the nicest custom-themed implementations of the https://github.com/okta/okta-signin-widget[Okta Sign-In Widget] we've seen to date, and makes use of self-registration and MFA flows. The application lets you add recurring bills, expenses, and budgets, and create projections.

Reviewer: ```suggestion
Bill Panda is one of the nicest custom-themed implementations of the https://github.com/okta/okta-signin-widget[Okta Sign-In Widget] we've seen to date, and makes use of self-registration and MFA flows. The application lets you add recurring bills, expenses, and budgets, and create projections.
```
two words

Marco Ledesma made Bill Panda to satisfy a personal itch. Learning about Okta auth has given him ideas for new features, and other ways to extend his project. We can't wait to see what's next for the panda, and look forward to hearing about the Laravel package mentioned by Marco for integrating Okta.

++++
<div style="text-align: center; margin-bottom: 1.25rem">
<iframe width="700" height="394" style="max-width: 100%" src="https://www.youtube.com/embed/N3O5F5EaHE8" title="Okta Hackathon Demo Video of Bill Panda" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
</div>
++++

== First Place

**Name:** https://devpost.com/software/gooseclip[GooseClip] +
**Prize:** $10,000 USD +
**Description:** Authenticated peer-to-peer communication platform

> Matthew Mckenzie's GooseClip submission was in a league of its own–featuring a highly polished demo video, as well as web, CLI, and mobile applications that demonstrated aspects of the project as well as its creator's technical prowess. Rooted in the Go programming language, and inspired by "a frustration at the lack of interoperability of our modern devices" GooseClip "integrates fast, secure peer-to-peer functionality" into a project in a few simple steps. It obscures the complexity of peer-to-peer networking by presenting a 2-route API, /attach and /connect.

Reviewer: ```suggestion
Matthew Mckenzie's GooseClip submission was in a league of its own. The project features a highly polished demo video, as well as web, CLI, and mobile applications that demonstrate aspects of the project as well as its creator's technical prowess. Rooted in the Go programming language, and inspired by "a frustration at the lack of interoperability of our modern devices," GooseClip "integrates fast, secure peer-to-peer functionality" into a project in a few simple steps. It obscures the complexity of peer-to-peer networking by presenting a 2-route API, /attach and /connect.
```
tweaked for readability.

Matthew has big plans to evolve GooseClip, and we plan to follow along. Here's hoping the grand prize will help this goose take flight!

++++
<div style="text-align: center; margin-bottom: 1.25rem">
<iframe width="700" height="394" style="max-width: 100%" src="https://www.youtube.com/embed/KLOtyv_yDSA" title="GooseClip - Okta Hackathon Submission" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
</div>
++++

== Most Supportive Hackers

There are many roads to glory, and all developers rely on the work of those who came before them. Sometimes help comes in the form of code—libraries, building blocks, open source projects. Other times, good information and solid guidance at the right time can mean the difference between failure and success.

> We awarded the "most supportive hacker" prize to three individuals, based on the number of valuable and helpful responses they provided to other participants on the Okta Developer Forum's hackathons category during the Hackathon submission period.

Reviewer: ```suggestion
We awarded the "most supportive hacker" prize to three individuals, based on the number of valuable and helpful responses they provided to other participants on the Okta Developer Forum's *hackathons* category during the competition.
```
punctuation tweak

> Let's close with kudos to Lenya Hope (1st place), Monish Basaniwal (2nd place), and Chaitanya Shah  (3rd place) – who didn't let competition get in the way of generous collaboration and knowledge-sharing. Thank you all.

Reviewer: ```suggestion
Let's close with kudos to Lenya Hope (1st place), Monish Basaniwal (2nd place), and Chaitanya Shah (3rd place) – who didn't let competition get in the way of generous collaboration and knowledge-sharing. Thank you all.
```
removed extra space
